incubator/foundry-vtt/templates/secrets.yaml

```diff
@@ -9,7 +9,11 @@ metadata:
     heritage: "{{ .Release.Service }}"
 type: Opaque
 data:
-  adminKey: {{ default "admin" .Values.foundryvtt.adminKey | b64enc }}
+  {{- if .Values.foundryvtt.adminKey }}
+  adminPassword: {{ default "admin" .Values.foundryvtt.adminKey | b64enc }}
```

Comment:
Can we use a  ternary here to improve readability?

`adminPassword: {{ default "admin" ( ternary .Values.foundryvtt.adminPassword .Values.foundryvtt.adminKey .Values.foundryvtt.adminPassword) | b64enc }}`

I think this would work because nulls are evaluated as False. This ternary should not only use `adminKey` if it's set but will also prefer `adminPassword` if both are set. This should encourage adoption of the new usage.